testplan/common/utils/sockets/client.py

```diff
@@ -37,49 +40,48 @@ def __init__(self, host, port, interface=None):
         self._timeout = None
 
     @property
-    def address(self):
+    def address(self) -> Tuple[str, int]:
         """
         Returns the host and port information of socket.
         """
         return self._client.getsockname()
 
     @property
-    def port(self):
+    def port(self) -> Union[str, int]:
         return self._input_port
 
-    def connect(self):
+    def connect(self) -> None:
         """
         Connect client to socket.
         """
         self._client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
The document says:

socket.bind(address)
Bind the socket to address. The socket must not already be bound. (The format of address depends on the address family — see above.)

A pair (host, port) is used for the [AF_INET](https://docs.python.org/3/library/socket.html#socket.AF_INET) address family, where host is a string representing either a hostname in internet domain notation like 'daring.cwi.nl' or an IPv4 address like '100.50.200.5', and port is an integer.

Because we are always create to AF_INET type of socket, i wonder if we really need to support bytes type of input.

Reply:
fixed